client/app/lib/constants/sharedConstants.ts

```diff
@@ -61,3 +61,8 @@ export default {
   STAFF_ROLES,
   AVAILABLE_LOCALES,
 };
+
+export const SUPPORT_EMAIL = 'coursemology@gmail.com' as const;
```

Comment:
I am thinking to put all these configurable (imagine when nusri wants to change these next time) as env constants. For locale and time zone, if the values do not exists in the env file, then can fall back to `en` and `Asia/Singapore`.

Reply:
I don't think environment variables are the right place to put these constants. Environment variables are ephemeral. I'd rather put these in `package.json`. That's the best place to put it since these values are not ephemeral, but per-build-based.

Then again I don't think NUSRI will want to use our client app.

Reply:
What do you mean by environment variables being ephemeral? So lets say if the support email is changed, or someone else uses our client, the app needs to be rebuilt and redeployed every time?

Reply:
Yes. They should change those values and commit it to their version control.